tests/unit/test_devices.py

```diff
@@ -47,8 +61,25 @@ def test_init(self, adb_connect):
         assert device.root_unplug_file == 'test/file'
         assert device.root_unplug_value == '0'
         assert device.root_unplug is True
+        assert device.power_device == 'fake_path'
+        assert device.device_settings_reqs == {'app1': ['a, b'], 'app2': ['c']}
         adb_connect.assert_called_once_with(device_id)
 
+    @patch('AndroidRunner.Adb.configure_settings')
+    @patch('logging.Logger.info')
+    def test_configure_settings_device(self, logger, configure_settings, device_with_app_settings):
+        device_with_app_settings.configure_settings_device("app1")
+        logger.assert_called_with('Enabling setting_1')
+        configure_settings.assert_called_with(device_with_app_settings.id, "setting_1",True)
+
+        device_with_app_settings.configure_settings_device("app2")
+        logger.assert_called_with('Enabling setting_2')
+
+        device_with_app_settings.configure_settings_device(None)
+        device_with_app_settings.configure_settings_device("app3")
+
+        configure_settings.call_count == 3
```

Comment:
shouldn't this line of code be something like `assert configure_settings.call_count == 3`?

Reply:
Indeed.  Whoops.